sentinel-core/log/block/block_log_task.cc

```diff
@@ -0,0 +1,82 @@
+#include "sentinel-core/log/block/block_log_task.h"
+
+#include <iostream>
+#include <memory>
+#include <thread>
+
+#include "sentinel-core/log/log_base.h"
+#include "sentinel-core/utils/time_utils.h"
+
+#include "absl/strings/str_format.h"
+#include "absl/synchronization/mutex.h"
+#include "absl/time/time.h"
+#include "spdlog/sinks/rotating_file_sink.h"
+
+using namespace Sentinel::Utils;
+
+namespace Sentinel {
+namespace Log {
+
+BlockLogTask::BlockLogTask() {
+  auto filename = LogBase::GetLogBaseDir() + kBlockLogFilename;
+  try {
+    logger_ = spdlog::rotating_logger_mt(kBlockLoggerName, filename,
+                                         kDefaultBlockLogMaxSize, 3);
+    logger_->set_pattern("%v");
+  } catch (const spdlog::spdlog_ex& ex) {
+    std::cerr << "Sentinel block log initialization failed: " << ex.what()
```

Comment:
If the logger to create failure, whether there is a need to start a thread?

Reply:
I've added checking in `Start()` method. If the logger is absent, the daemon thread won't start.